### ISSUE TITLE
Can't drag the thumb forward or backward, but can click on the bar of slider, when testing on iPhone's Safari.

### ISSUE THREAD

**Issue by KaygNas** [microsoft/fluentui#17706]
### Environment Information

- **Package version(s)**: 0.53.0
- **Browser and OS versions**: safari, iOS 14.4.2

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/sangyaaak/pen/eYgEgJb

#### Actual behavior:

Can't drag the thumb forward or backward, but can click on the bar of slider, when testing on iPhone's Safari.
Everything normal when testing on Edge Device Emulation.

#### Expected behavior:

Should able to drag the thumb on Mobile Phone.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Low

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
@KaygNas - Thanks for reaching out to us. 
@kolaps33 - Would you be able to confirm if this is a regression, or if this behavior is an issue when using Safari? I checked on chrome and the behavior was correct.

**Comment by KaygNas** [comment]
> @KaygNas - Thanks for reaching out to us.
> @kolaps33 - Would you be able to confirm if this is a regression, or if this behavior is an issue when using Safari? I checked on chrome and the behavior was correct.

Yes, the incorrect behavior only appear when using Safari, as I open the codepen  on my iPhone and it behave as the same as my description above.
